components/collapsible/accordion/src/index.js

```diff
@@ -0,0 +1,54 @@
+import React, {Component, PropTypes} from 'react'
+import CollapsibleBasic from '@schibstedspain/sui-collapsible-basic'
+
+class CollapsibleAccordion extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {items: props.items}
+    this._handleClick = this._handleClick.bind(this)
+    this._collapseItems = this._collapseItems.bind(this)
+  }
+
+  _handleClick (id) {
+    return (collapsed) =>
+      this._collapseItems(collapsed, id)
+  }
+
+  _collapseItems (collapsed, id) {
+    const items = this.state.items
+    this.setState({items: items.map((item, index) => {
+      item.collapsed = (index === id) ? collapsed : true
+      return item
+    })})
+  }
+
+  _renderCollapsible () {
+    const { items } = this.state
+    return items.map((item, index) => {
+      return (
+        <CollapsibleBasic key={index} handleClick={this._handleClick(index)} collapsed={item.collapsed} label={<div style={{padding: '8px'}}>{item.label}</div>} >
```

Comment:
Usually we put one props per line for readability.
Please never style with `style` prop. Use css classes. Do you really need this div ?
Id "div" the most semantic tag? Perhaps <p> of <section> would be more SEO friendly.

Also, to avoid breaking API in a future, you can do:

```js
return items.map((item, index) => {
      return (
        <CollapsibleBasic {...item} key={index} handleClick={this._handleClick(index)} >
```
Then if in the future CollapsibleBasic has more props, you won't have to change this component to make it work.